Add example and update readme for Login Plugin (#3)
deephaven/js-plugin-template: README.md

## Setting an Application Level Plugin

In the config set `Webapi.app.plugins` to your plugin name. This can be a comma separated list to support multiple plugins.

## Login Plugin

> A Login Plugin loads before the user log into the Web Client, and it allows UI permissions to be set. See `ExampleLoginPlugin.jsx` for an example.

Reviewer: ```suggestion
A Login Plugin loads before the user logs in to the Web Client, and it allows UI permissions to be set. See `ExampleLoginPlugin.jsx` for an example.
```

Author: Fixed

The current supported permissions are:

- canUsePanels - Allows the user access to the `Panels` button in the UI.
- canCreateDashboard - Allows the user to create a new Dashboard.
- canCreateCodeStudio - Allows the user to create a new Code Studio.

> - canCreateQueryMonitor - Allows the user to open the Query Monitor.

Reviewer: May as well keep the language consistent
```suggestion
- canCreateQueryMonitor - Allows the user to create a new Query Monitor.
```

Author: Fixed
